linera-views/src/common.rs

```diff
@@ -41,6 +42,16 @@ pub(crate) enum Update<T> {
 /// The minimum value for the view tags. Values in 0..MIN_VIEW_TAG are used for other purposes.
 pub(crate) const MIN_VIEW_TAG: u8 = 1;
 
+/// Data type indicating that the database is not coherent
```

Comment:
nit: "consistent" is more common in the world of databases in English. Not sure if it's the right concept though.

Reply:
Ok, renamed.